measured-rails.gemspec

```diff
@@ -13,15 +13,16 @@ Gem::Specification.new do |spec|
   spec.homepage      = "https://github.com/Shopify/measured-rails"
   spec.license       = "MIT"
 
-  spec.files         = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
-  spec.bindir        = "exe"
-  spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
+  spec.files         = `git ls-files -z`.split("\x0")
+  spec.executables   = spec.files.grep(%r{^bin/}) { |f| File.basename(f) }
+  spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
```

Comment:
This doesn't really change anything, but it brings it in line with the same file in measured which was generated with a slightly different bundler version.